specification/dimensions/usageunit.md

```diff
@@ -33,18 +43,37 @@ Units listed in the following table are presented in the following format: "abbr
 
 If the unit is not listed in the table, it is to be used over a functional equivalent with similar meaning or incompatible capitalization.  An example of unit now allowed is a unit such as "calls" or "number" to signify the number of times an event occurred in this case, the unit "count" MUST be used to maintain compatibility with this specification.
```

Comment:
GitHub won't let me comment on the other section since it's from a previous commit 😒

Can we label this section "Allowed values" and make it an H3, like the other normalized columns?